operator/internal/handlers/internal/storage/secrets.go

```diff
@@ -129,12 +125,12 @@ func extractS3SSEConfig(d map[string][]byte) (storage.S3SSEConfig, error) {
 		kmsKeyId, kmsEncryptionCtx string
 	)
 
-	switch sseType = storage.S3SSEType(d["sse_type"]); sseType {
+	switch sseType = storage.S3SSEType(d[storage.KeyAWSSSEType]); sseType {
 	case storage.SSEKMSType:
-		kmsEncryptionCtx = string(d["sse_kms_encryption_context"])
-		kmsKeyId = string(d["sse_kms_key_id"])
+		kmsEncryptionCtx = string(d[storage.KeyAWSSSEKMSEncryptionContext])
+		kmsKeyId = string(d[storage.KeyAWSSSEKMSKeyID])
```

Comment:
I do know that these are all valid acronyms written together, but this _is_ an ugly constant name.